recipes/flatbuffers/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: flatbuffers
+  version: {{ version }}
+
+source:
+  fn: flatbuffers-{{ version }}.tar.gz
+  url: https://github.com/google/flatbuffers/archive/v{{ version }}.tar.gz
+  md5: 86d13e8c4c19c2bbadff86c5435b2fca
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+
+test:
+  commands:
+    - test -f $PREFIX/bin/flatc
```

Comment:
Is there a help or version command that we can run with `flatc`? While running such a simple command like help or version is not a great test, it is good at attempt to load up any relevant dynamic libraries. If this fails, we know we need to go back and handle linking errors. If not, we can be pretty confident it linked together ok.